components/src/status_im/ui/components/react.cljs

```diff
@@ -94,9 +94,13 @@
 
 (def pan-responder (lazy-get-react-property "PanResponder"))
 (def animated (lazy-get-react-property "Animated"))
-(defn animated-view []
+
+(defn animated-view-class []
   (reagent/adapt-react-class (.-View (animated))))
 
+(defn animated-view [props content]
+  [animated-view-class props content])
```

Comment:
@rasom is it correct here I wasn't sure about this one it's a bit different than the other